### ISSUE TITLE
Model Architecture and Model Training Approach

### ISSUE THREAD

**Issue by Bempong-Sylvester-Obese** [Bempong-Sylvester-Obese/RSNA-Intracranial-Aneurysm-Detection#1]
* Model Architecture should be a 3D U-Net.
* Model Architecture should be loaded before the loss function.
* After the loss function, training functions have to be implemented.
* After the training functions have been loaded, the next step is to work on the main training loop.
* Next step is to run the training pipeline.
* Evaluate Epoc.
